apps/root/src/pages/aggregator/swap-container/components/swap/index.tsx

```diff
@@ -1180,16 +1181,6 @@ const Swap = ({ isLoadingRoute, quotes, fetchOptions, swapOptionsError }: SwapPr
         onCancel={() => setShouldShowTransferModal(false)}
         open={shouldShowTransferModal}
       />
-      <BetterQuoteModal
-        selectedRoute={selectedRoute}
-        betterQuote={betterQuote}
-        onCancel={() => setShouldShowBetterQuoteModal(false)}
-        onGoBack={handleBackTransactionSteps}
-        open={shouldShowBetterQuoteModal}
-        onSelectBetterQuote={(response: BlowfishResponse) =>
-          handleTransactionSimulationWait(transactionsToExecute, response)
-        }
-      />
       <FailedQuotesModal
```

Comment:
We should also remove this one right? Notification should be same for both

Reply:
I wasn't sure how to handle this case. now we open the modal and when it closes, we send the user back to first step.
I wouldn't to the same as this "Better quote" notification, the user would be blocked. We could close the txSteps and render an error notification in the snackbar?